qemu/tests/cfg/fio_windows.cfg

```diff
@@ -1,16 +1,31 @@
 - fio_windows:
     type = fio_windows
     only Windows
+    images += " stg"
+    image_size_stg = 2G
+    image_name_stg = images/stg
+    create_image_stg = yes
+    force_create_image_stg = yes
     cmd_timeout = 600
-    log_file = "C:\fio_log.txt"
     image_snapshot = yes
+    disk_index = 1
```

Comment:
what's disk_index for?why not 2?

Reply:
Hi lijin, 

the disk_index =1 means the disk number, for system disk the default number is 0, we see it as "Disk 0" in guest disk management,  if we add another disk to the windows guest, system will number this disk to "Disk 1" . Here, the disk_index =1 will be transferred to format_disk program. make sure the format operation on data disk (Disk 1).